protos/protos/public/modeldb/versioning/VersioningService.proto

```diff
@@ -669,4 +703,18 @@ service VersioningService {
             }
         };
     }
+
+    rpc findRepositories(FindRepositories) returns (FindRepositories.Response){
+        option (google.api.http) = {
+            post: "/v1/versioning/findRepositories"
+            body: "*"
+        };
+    };
+
+    rpc findRepositoriesBlobs(FindRepositoriesBlobs) returns (FindRepositoriesBlobs.Response){
+        option (google.api.http) = {
+            post: "/v1/versioning/findRepositoriesBlobs"
```

Comment:
```suggestion
            post: "/v1/versioning/findRepositoriesBlobs"
            post: "/v1/versioning/workspaces/{repository_id.named_id.workspace_name}/repositories/{repository_id.named_id.name}/findRepositoriesBlobs"
            additional_bindings {
                post: "/v1/versioning/repositories/{repository_id.repo_id}/findRepositoriesBlobs"
            }
```

Reply:
@ravishetye we have `repeated RepositoryIdentification repository_id = 1;` is repeated so we can not add single `{repository_id.repo_id}` OR `{repository_id.named_id.name}` in url. if this filter only with respected to repository then it could be works.

Reply:
updated